docusaurus/docs/reference/prompts.md

```diff
@@ -49,3 +49,27 @@ Give your plugin a name which helps identify its purpose.
 ### Enter your organization name (usually your Grafana Cloud org)
 
 Enter the name of your organization. This must be your [Grafana Cloud](https://grafana.com/signup/) organization to allow you to [sign](../publish-a-plugin/sign-a-plugin.md) and optionally [publish](../publish-a-plugin/publish-or-update-a-plugin.md) the plugin to the [catalog](https://grafana.com/grafana/plugins).
+Add a [GitHub workflow](/create-a-plugin/develop-a-plugin/set-up-github-workflows#the-compatibility-check-is-compatibleyml) to regularly check that your plugin is compatible with the latest version of Grafana.
```

Comment:
shouldn't be here?

Reply:
Ah sorry I don't think I was paying attention to the merge conflicts!